### DISCUSSION TITLE
Privacy policy clarification on data collection

### DISCUSSION THREAD

**Discussion by strangeh21** [rustdesk/rustdesk#4018] [Ideas]
Hi, I'd like some clarification on the privacy policy regarding data collection under category d. RustDesk: [https://rustdesk.com/privacy/](https://rustdesk.com/privacy/)

> To provide you with the RustDesk software application and to constantly improve our services including customer support, we process following personal data about you and your device: start of the RustDesk software application, IP-address of the device, statistical information about your computer (e.g. CPU-type, screen resolution), time and duration of RustDesk software sessions and RustDesk-IDs of the RustDesk’s session participants.
> 
> We process the personal data acquired via the RustDesk software under lawful grounds of performance of a contract (performance of RustDesk software license agreement concluded between us and yourself including customer support) and our legitimate interest (RustDesk software development).

After some discussion in Discord it was assumed that only the relay server is collecting this data, and that there's no other data collection outside of this. In which case, those using their own relay server there is no form of telemetry what so ever, except that which is collected by their own relay server. 
I wanted this clarified and if it is true, I'd like to request that the privacy policy is edited to reflect this.

**Comment by rustdesk** [comment]
Honestly, the privacy policy file was copied from somewhere when i started the project. Do need to refine. Thanks for your reminder.

**Reply by anhofmann** [reply]
Well, uh, I would like to remind you about that. Its been three and a half years now.

**Comment by simeononsecurity** [comment]
A valid privacy policy should be defined. 
For this to be a truly open source project, any information that is collected should be documented and the client and server side clients should have the ability to disable said telemetry if they choose.
If this project is ever going to be profitable, which will require enterprise buy in, you need to have support for GDPR, HIPPA, FIPS 140-2, PCI DSS, etc compliance. Proper data handling and the ability to disable telemetry is a absolute must. @rustdesk

**Comment by Nemo7CZ** [comment]
This is preventing me from using RustDesk. I can't install it with good concise on other peoples machines until analytics is opt in. My support does not come with reporting to a 3rd party.

Edit: The "This app tracks and reports your activity" anti-feature on F-droid also isn't a good look.

**Reply by xlionjuan** [reply]
The default Public Server is treat as "telemetry" on F-Droid, but it actually not.

https://github.com/rustdesk/rustdesk/discussions/10095

**Reply by xlionjuan** [reply]
Or the check update API": https://github.com/rustdesk/rustdesk/discussions/11055

**Reply by Nemo7CZ** [reply]
> Or the check update API": #11055

That is basically telemetry though. It should not happen by default.

**Reply by Nemo7CZ** [reply]
> The default Public Server is treat as "telemetry" on F-Droid, but it actually not.
> 
> #10095

Thanks for confirming that. I still think there should be a dialog that you have to accept on first connection about the data collection by the default server and an ability to just keep it disabled if the user wishes.

**Reply by LoganJFisher** [reply]
> This is preventing me from using RustDesk. I can't install it with good concise on other peoples machines until analytics is opt in. My support does not come with reporting to a 3rd party.
> 
> Edit: The "This app tracks and reports your activity" anti-feature on F-droid also isn't a good look.

"This app tracks and reports your activity" "Connects to the dev site sending date of start, IP of device, device info, time and duration of sessions, and RustDesk-IDs"

I agree. This makes me extremely hesitant. I just don't see why this should be necessary at all for what RustDesk offers.

**Comment by xlionjuan** [comment]
@ rustdesk consider put check update endpoint (I guess it is just some kind of JSON, but I didn't sure) to GH pages, this "might" help dispel concerns.

**Comment by duaneking** [comment]
I see no reasonable reason  you need to collect my IP address for me to connect from one computer on my local network to connect to another computer on my local network, and to be honest I'm worried this is spyware that is not trustworthy because of this.

**Reply by southerndoc** [reply]
Replied in the other thread you started.  If you're that concerned about it, then don't use it.  Every app you install on your computer probably phones home and every website you visit gets your IP address.  Who cares?  I have a static IPv4/IPv6 address that never changes and I could care less if RustDesk developers get it.  Enable 2FA per device.  With strong passwords and 2FA per device, it's hard to break into your computer.

RustDesk obtaining telemetry data doesn't mean they are hacking into your computer to read your emails, make financial transactions, or view your webcam.  I think you're being overly paranoid about telemetry data that allows them to see ways to improve their product.

**Comment by duaneking** [comment]
The worries many people have is that they're collecting IP addresses to then target them or do some kind of other thing with them and that is not something that any of us want so the removal of the IP address is going to be critical for a lot of people to gain trust there is no reason for that collection or that telemetry at all and it just looks bad.

**Reply by southerndoc** [reply]
I think you're being a conspiracy theorist here.  Other remote desktopVNC software I use phones home.  Try Jump Desktop (that won't function without contacting their servers), RealVNC (also phones home to verify your subscription), etc.  They all obtain your IP address.  Doesn't mean they're looking for ways to hack into your system.

Enable a strong password, set the setting so that only permanent passwords are allowed (don't allow one-time passwords unless you truly need it -- I set my password to >64 characters), enable 2FA per device and not just your account, self-host of course, and use IP whitelisting.  Those things make it pretty impenetrable.  Doesn't mean it's impossible.  However, it makes it very highly unlikely someone will break into your system.

I've yet to see anything that offers the degree of security that RustDesk offers.  I went away for a bit after their developer ticked me off with a rude comment, but ultimately came back because of the security of it.
